tests/data/empty_machineconfig.json

```diff
@@ -0,0 +1,4 @@
+{
+  "metadata": {},
+  "items": null
```

Comment:
It is weird that on 4.16, hipershift, through CLI I get `"items": []`, while the API seems to get `"items": null`.

The cli `gojq` behaves consistent with the errors we are facing.


```console
$ echo '{"metadata": {}, "items": []}' | gojq '[.items[] | select(.metadata.name | test("^rendered-worker-[0-9a-z]+$|^rendered-master-[0-9a-z]+$"))] | map(.spec.fips == true)'
[]
```
```console
$ echo '{"metadata": {}, "items": null}' | gojq '[.items[] | select(.metadata.name | test("^rendered-worker-[0-9a-z]+$|^rendered-master-[0-9a-z]+$"))] | map(.spec.fips == true)'
gojq: cannot iterate over: null
```

Reply:
Yeah - that seems like an API change in the machine config API. Maybe there was a alpha to beta, or beta to stable jump between those versions.